redisinsight/api/src/modules/recommendation/providers/recommendation.provider.ts

```diff
@@ -411,6 +416,122 @@ export class RecommendationProvider {
     }
   }
 
+  /**
+  * Check search indexes recommendation
+   * @param redisClient
+   * @param keys
+   * @param client
+   */
+  // eslint-disable-next-line
+  async determineSearchIndexesRecommendation(
+    redisClient: Redis,
+    keys: Key[],
+    client: any,
+  ): Promise<Recommendation> {
+    try {
+      if (client.isCluster) {
```

Comment:
Please move this to separate method. Let's have 2 separate method determineForCluster*... and determineForStandalone*...